### DISCUSSION TITLE
Check for updates!

### DISCUSSION THREAD

**Discussion by kristiker** [d2phap/ImageGlass#1105] [General] [answered]
I'm sorry but the auto update popup is super super annoying. No one cares if their image viewer is out of date thanks. I know i can disable.

**Comment by d2phap** [comment, marked as answer]
Hi, I don't get what you are trying to ask here?
You can disable the Check for update in Settings, so it should be fine to you.

**Reply by kristiker** [reply]
It is fine now, but it has wasted me so much time, when all I wanted was to quickly open an image file. I suggest looking at how other applications (operating systems, graphics drivers, text editors, media players) deal with being out of date.

**Comment by victorczon** [comment]
I can agree that it is annoying, it would be awesome if it had auto update feature
